### PR TITLE
Add user interview with Europace

### PR DESCRIPTION
**Pull request by brian-brazil**

### CODE REVIEW

File: content/blog/2017-04-06-interview-with-europace.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: Interview with Europace
+created_at: 2017-04-06
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, Tobias Gesellchen from
+Europage talks about how they discovered Prometheus.*
```
Comment (juliusv) [inline content/blog/2017-04-06-interview-with-europace.md]: Europage -> Europace

File: content/blog/2017-04-06-interview-with-europace.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+demand for flexibility we looked for other solutions. Due to Nagios and Icinga
+being more centrally maintained, Prometheus matched our aim to have the full
+DevOps stack in our team and move specific responsibilities from our
+infrastructure team to the project members.
+
+## Why did you decide to look at Prometheus?
+
+Through our activities in the [Docker Berlin
+community](https://www.meetup.com/Docker-Berlin/) we had been in contact with
+[SoundCloud] (https://soundcloud.com/) and [Julius
```
Comment (juliusv) [inline content/blog/2017-04-06-interview-with-europace.md]: Link is broken due to space between link text and URL.

File: content/blog/2017-04-06-interview-with-europace.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Volz](https://twitter.com/juliusvolz), who gave us a good overview. The
+combination of flexible Docker containers with the highly flexible labels based
```
Comment (juliusv) [inline content/blog/2017-04-06-interview-with-europace.md]: "labels based" -> "label-based"

File: content/blog/2017-04-06-interview-with-europace.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+concept convinced us give Prometheus a try.  The Prometheus setup was easy
+enough, the Alertmanager worked for our needs, so that we didn’t see any reason
```
Comment (juliusv) [inline content/blog/2017-04-06-interview-with-europace.md]: run-on sentence / missing conjunction:

s/the Alertmanager/and the Alertmanager/

**Comment by juliusv** [comment]
:+1: otherwise

**Comment by brian-brazil** [comment]
Updated, thanks.

**Comment by juliusv** [comment]
:+1:

**Comment by gesellix** [comment]
thx @juliusv

**Pull request merged**